build/Dockerfile.checkov

```diff
@@ -6,6 +6,7 @@ ARG CHECKOV_VERSION
 ENV CHECKOV_VERSION="${CHECKOV_VERSION}"
 ENV CHECKOV_JSON_REPORT_PATH="/tmp/reports/checkov"
 ENV SKIP_CHECKOV="false"
+ENV CHECKOV_LOG_LEVEL="WARNING"
```

Comment:
This and the Dockerfrag comes from https://github.com/bridgecrewio/checkov/blob/74dc72b76b22b82f46e38263158f1393f1d68bab/checkov/logging_init.py#L7